scraper/spec-urls.txt

```diff
@@ -0,0 +1,279 @@
+http://www.w3.org/TR/mixed-content/
```

Comment:
Would it make sense to keep this list in alphabetical order?

Reply:
It's already in alphabetical order

Reply:
lol